avm/utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -203,24 +203,21 @@ Describe 'Module tests' -Tag 'Module' {
           if (-not $templateContent) {
             throw ($bicepTemplateCompilationFailedException -f $templateFilePath)
           }
-        }
-        elseIf (Test-Path (Join-Path $moduleFolderPath 'main.json')) {
+        } elseIf (Test-Path (Join-Path $moduleFolderPath 'main.json')) {
```

Comment:
I'll take this after #606 to avoid ghost changes and all modules pipeline trigger :ghost:

Reply:
I'd appreciate that. It's somewhere in my main and hard to get rid of